app/constants.py

```diff
@@ -268,7 +268,16 @@ class OrganisationUserPermissionTypes(enum.StrEnum):
 # Service callbacks
 DELIVERY_STATUS_CALLBACK_TYPE = "delivery_status"
 COMPLAINT_CALLBACK_TYPE = "complaint"
-SERVICE_CALLBACK_TYPES = [DELIVERY_STATUS_CALLBACK_TYPE, COMPLAINT_CALLBACK_TYPE]
+RETURNED_LETTERS_CALLBACK = "returned_letters"
+SERVICE_CALLBACK_TYPES = [DELIVERY_STATUS_CALLBACK_TYPE, COMPLAINT_CALLBACK_TYPE, RETURNED_LETTERS_CALLBACK]
```

Comment:
Are we going to replace these with the `enum` at some point? Can we do that in this PR?

Reply:
That is the plan. However I will remove the code for the enum from this PR and deal with it in a separate PR.